static/monitoring/package.json

```diff
@@ -3,27 +3,19 @@
     "version": "0.0.1",
```

Comment:
This is just a special file to teach Gemnasium about libraries that we use but don't install cleanly.